.github/workflows/build.yaml

```
@@ -0,0 +1,222 @@
+
+      - name: Save Database Docker Image
+        id: save_db_image
+        if: ${{ steps.check-db-folder.outputs.folder_exists == 'true' }}
+        run: |
+          IMG="egovio/${{ matrix.service }}-db:${{ steps.tag.outputs.tag }}"
+          echo "$IMG"
+          echo "db_image=$IMG" >> "$GITHUB_OUTPUT"
+
+          # Append to GitHub Actions summary
+          echo "### Database Docker Image" >> "$GITHUB_STEP_SUMMARY"
+          echo "$IMG" >> "$GITHUB_STEP_SUMMARY"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)